pkg/cmd/config.go

```diff
@@ -610,6 +610,9 @@ containerRuntime:
   #     docker.io:
   #       mirrors:
   #       - http://plain-text2.tld
+  #       auth:
+  #         username: "uzer"
+  #         password: "myc00lp455w0rd"
```

Comment:
Can we also document Auth and IdentityToken fields?

Reply:
The docs are in the generated API, this is just an example. But nether the less, done.